src/connector/src/parser/unified/debezium.rs

```diff
@@ -204,12 +205,18 @@ pub fn parse_schema_change(
 
                     let data_type = match *connector_props {
                         ConnectorProperties::PostgresCdc(_) => {
-                            DataType::from_str(type_name.as_str()).map_err(|err| {
-                                tracing::warn!(error=%err.as_report(), "unsupported postgres type in schema change message");
-                                AccessError::UnsupportedType {
-                                    ty: type_name.clone(),
+                            let ty = type_name_to_pg_type(type_name.as_str());
+                            match ty {
+                                Some(ty) => pg_type_to_rw_type(&ty).map_err(|err| {
+                                    tracing::warn!(error=%err.as_report(), "unsupported postgres type in schema change message");
+                                    AccessError::UnsupportedType {
+                                        ty: type_name.clone(),
+                                    }
+                                })?,
+                                None => {
+                                    Err(AccessError::UnsupportedType { ty: type_name })?
```

Comment:
Did you intend to split these 2 different error cases? Why only logging in the first case?